src/doc/src/reference/manifest.md

```diff
@@ -589,6 +589,22 @@ Cargo only applies these to the current package and not to dependencies.
 As for dependents, Cargo suppresses lints from non-path dependencies with features like
 [`--cap-lints`](../../rustc/lints/levels.html#capping-lints).
 
+### Special `check-cfg` sub-config for the `unexpected_cfgs` lint {#special-check-cfg}
+
+The `unexpected_cfgs` Rust lint accepts a special sub-config named `check-cfg` that is
+a list of compiler arguments to be passed, see the documentation of
+[`cargo::rustc-check-cfg`][rustc-check-cfg] for more details.
+
+This special sub-config is only allowed for `[lints.rust.unexpected_cfgs]`, it can be
+used like this:
+
+```toml
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = ["cfg(tokio_unstable)"] }
+```
+
+[rustc-check-cfg]: build-scripts.md#rustc-check-cfg
```

Comment:
Are we expected to be documenting rustc's lint configs or should we be pointing to rustc's docs (once they are updated)?

Reply:
I've also removed this documentation update.